_sass/_base.scss

```
@@ -461,9 +461,13 @@ footer.sticky-bottom {
 
     li {
       margin-bottom: 1rem;
+      .preview {
+        width: 100%;
```

some styles for better responsiveness
```suggestion
        width: 100%;
        min-width: 80px;
        max-width: 200px;
```